bitnami/mongodb/templates/replicaset/statefulset.yaml

```diff
@@ -531,6 +531,11 @@ spec:
           emptyDir: {}
           {{- end }}
   {{- else }}
+  {{- if .Values.persistentVolumeClaimRetentionPolicy.enabled }}
+  persistentVolumeClaimRetentionPolicy:
+    whenDeleted: {{ .Values.persistentVolumeClaimRetentionPolicy.whenDeleted }}
+    whenScaled: {{ .Values.persistentVolumeClaimRetentionPolicy.whenScaled }}
+  {{- end }}
```

Comment:
I guess you should be adding the same modifications to `mongodb/templates/standalone/dep-sts.yaml`. At the end of the day, it is another statefulset that is used when another deployment option is selected.

Reply:
makes perfect sense!
just updated

Reply:
Good catch!